### ISSUE TITLE
[4.x]: Field layout deletes empty tabs on save

### ISSUE THREAD

**Issue by ccchapman** [craftcms/cms#13132]
### Description

When saving an entry type, tabs without fields are lost.

I often find myself creating a bunch of empty tabs to scaffold an entry type.  It is frustrating that the tabs are irrecoverably lost.

### Steps to reproduce

1. Edit an entry type
2. Add a new tab without elements
3. Click save

### Expected behavior

Empty tab is not lost.  This could mean that the tab shows a validation error ("Tabs must have at least one element.") or empty tabs are persisted without error.

### Actual behavior

Empty tab is lost.

### Craft CMS version

4.4.8

### PHP version

8.2

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
This is currently working as intended. Why create a tab you’re intentionally leaving blank?

**Comment by ccchapman** [comment]
> This is currently working as intended. Why create a tab you’re intentionally leaving blank?

If I am creating an entry type, I often create a bunch of blank tabs to scaffold how the entry type should look.  If you instinctively save the entry type before populating the elements on the tab, those tabs disappear with no recourse.

**Comment by brandonkelly** [comment]
Yeah, there is some code in place that is intentionally removing unneeded tabs. We can discuss internally whether it makes sense to remove that.

**Comment by ccchapman** [comment]
> Yeah, there is some code in place that is intentionally removing unneeded tabs. We can discuss internally whether it makes sense to remove that.

👍 

There are some situations where you might expect this behavior.  Like if the tab is preexisting and you remove all elements from it.

I personally might lean towards having a double-confirm ("You are saving a field layout with blank tabs.  Blank tabs will be removed.  Continue?")  Maybe not a common convention in Craft but somewhat similar to the popup when deleting a user.

**Comment by brandonkelly** [comment]
I think if anything we would just stop removing them. It’s not a huge deal either way because blank tabs don’t get included in element editors (e.g. if all the fields within them are hidden due to conditions).
